src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Main.cs

```diff
@@ -294,6 +325,18 @@ private static Result CreateOpenCurrentFolderResult(string search)
             };
         }
 
+        private Result CreateTruncatedItemsResult(string search, int preTruncationCount, int postTruncationCount)
+        {
+            return new Result
+            {
+                Title = _context.API.GetTranslation("Microsoft_plugin_folder_truncation_warning_title"),
+                QueryTextDisplay = search,
+                SubTitle = string.Format(CultureInfo.InvariantCulture, _context.API.GetTranslation("Microsoft_plugin_folder_truncation_warning_subtitle"), postTruncationCount, preTruncationCount),
+                IcoPath = WarningIconPath,
+                Score = 1,
```

Comment:
Is the score set to 1 because we want the warning to follow the list of files and folders? I don't think setting the score to 1 is the best way to go about it because all the results of the folder plugin have a score of 500. There might be some program plugin results in between with a score > 0 which might be above the warning. Maybe we'd want to set it to 499 instead so that it is shown after all the folder plugin results but also ensuring that no other results appear in between. Thoughts?

Reply:
All the results from the folder plugin have score 0 as given [here](https://github.com/microsoft/PowerToys/blob/master/src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Main.cs#L244). And I couldn't find any place where the score was changed. So, I had accordingly set warning score to 1 to display these above all folder plugin results.

Reply:
All the results from the folder plugin have score 0 as given [here](https://github.com/microsoft/PowerToys/blob/master/src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Main.cs#L244). And I couldn't find any place where the score was changed. So, I had accordingly set warning score to 1 to display these above all folder plugin results.

Reply:
Sounds good. I got confused with the score of the open current folder which is 500, set [here](https://github.com/microsoft/PowerToys/blob/master/src/modules/launcher/Plugins/Microsoft.Plugin.Folder/Main.cs#L288). Thanks!

Just thinking out loud, is it better to show the warning at the end, only to the users who actually scroll down to find a result that has been truncated?

Reply:
IMO is better to show a warning before so that the user can get an idea that results are truncated and he can use a more specific query rather than scrolling 100 results and then finding a warning. 

@saahmedm @crutkas can take a call on whether we want to display a warning at the end or start.

Reply:
Sure. It doesn't have to block the PR.

Reply:
imo warning should be at the bottom, its confusing at the top as it worries me using it that all the results may not be there. whereas for users who care to scroll, its a clear explanation.